packages/frontend-v2/utils/plan/preAndCoReqCheck.ts

```diff
@@ -1,96 +1,135 @@
 import {
-  courseError, INEUAndPrereq, INEUOrPrereq, INEUPrereq, INEUPrereqCourse, INEUPrereqError, preReqWarnings, Schedule2, ScheduleTerm2, courseToString
+  CoReqWarnings,
+  courseToString,
+  INEUAndPrereq,
+  INEUOrPrereq,
+  INEUPrereq,
+  INEUPrereqCourse,
+  INEUPrereqError,
```

Comment:
Since we're using them for both prereqs and coreqs, we should probably replace the word prereq with just req in the INEU*Prereq types for the sake of clarity. (I know these type names predate this PR but now is probably a good time to change them).

Reply:
True okay will amend

Reply:
True okay will amend